src/lib/ibcclient.ts

```diff
@@ -27,6 +27,7 @@ import {
   Tendermint34Client,
 } from '@cosmjs/tendermint-rpc';
 import { arrayContentEquals, assert, sleep } from '@cosmjs/utils';
+import cloneDeep from 'lodash/cloneDeep';
```

Comment:
Is it a crime to bring `lodash` only for one function?

Reply:
Not sure about a crime. If we didn't already have >1 MB dependencies, I would say it is too heavy. But I'll leave the call to will here

Reply:
@willclarktech

Reply:
I'm sure Will has a `reduce` command for that